packages/flutter/lib/src/services/font_loader.dart

```diff
@@ -4,7 +4,7 @@
 
 import 'dart:async';
 import 'dart:typed_data';
-import 'dart:ui';
+import 'dart:ui' hide VoidCallback;
```

Comment:
why are lines like this needed? I would have expected any case where dart:ui is imported, the only other imported VoidCallback would be the same one?

Reply:
I've removed all these cases. They were added as part of trying to debug the cause of the strange analysis errors. As you say, there is no need to ever hide `VoidCallback` when `dart:ui` is imported.